test/HealthChecks.AzureStorage.Tests/DependencyInjection/AzureBlobStorageRegistrationTests.cs

```diff
@@ -7,7 +7,7 @@ public void add_health_check_when_properly_configured()
         {
             var services = new ServiceCollection();
             services.AddHealthChecks()
-                .AddAzureBlobStorage("the-connection-string");
+                .AddAzureBlobStorage(connectionString: "the-connection-string");
```

Comment:
As you can see, these new methods cause ambiguity which is not desirable. I'm thinking about how to introduce this more cleanly, especially for existing users.

Reply:
Compile-time error in any case is more preferable then run-time error.

Reply:
The new method overloads now accept a delegate for configuring the options, so these method signatures are no longer ambiguous!